crypto.c

```diff
@@ -117,9 +118,20 @@ unsigned int field_eq(const Field a, const Field b)
     }
 }
 
-void scalar_copy(Scalar c, const Scalar a)
+void scalar_copy(Scalar b, const Scalar a)
 {
-    fiat_pasta_fq_copy(c, a);
+    fiat_pasta_fq_copy(b, a);
+}
+
+void scalar_from_words(Scalar a, const uint64_t words[4])
+{
+    uint8_t *p = (uint8_t *)words;
+    uint8_t bytes[sizeof(Scalar)];
```

Comment:
This function relies on word internal endianness right? It looks like it's not really needed -- AFAICT you really want scalar_from_bytes_le or something?

Can you change it to not rely on word-internal endianness? I know this is only used in unit tests so it is fine now, but it's important to keep this repo endianness-independent.

Reply:
Fixed!